app/src/main/java/de/westnordost/streetcomplete/quests/shop_type/CheckShopExistenceForm.kt

```diff
@@ -0,0 +1,23 @@
+package de.westnordost.streetcomplete.quests.existence
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.quests.AbstractOsmQuestForm
+import de.westnordost.streetcomplete.quests.AnswerItem
+
+class CheckShopExistenceForm : AbstractOsmQuestForm<Unit>() {
+    override val buttonPanelAnswers get() =
+        if (isAskingAboutExistence) {
+            listOf(
+                AnswerItem(R.string.quest_generic_hasFeature_no) {
+                    replaceShop()
+                },
+                AnswerItem(R.string.quest_generic_hasFeature_yes) {
+                    applyAnswer(Unit)
+                }
+            )
+        } else {
+            emptyList()
+        }
+
+    private var isAskingAboutExistence: Boolean = true
```

Comment:
What is this variable used for? I think it's never overwritten, so it could be removed, right?

Reply:
oh right, that is leftover from the initial design